### PR TITLE
[Fix] Portal minor fixes

### PR DESCRIPTION
**Pull request by pjamsheer**
## Feature description
- Portal labels are fixed

## Is there any existing behavior change of other features due to this code change?
No

## Was this feature tested on all the browsers?
  - [x] Chrome
  - [x] Safari

### CODE REVIEW

File: one_fm/templates/pages/career_history.html [hunk 1]
```diff
@@ -9,15 +9,15 @@
 {% endblock %}
 
 {% block header %}
-  <h5 id="job_applicant" data="{{ job_applicant.name }}">Hey {{job_applicant.applicant_name}} !,
-		You’re Applying for {{job_applicant.one_fm_designation}} Position.</h5>
-	<h5>Enter all the details about your career in below form and tell us How great you’re!</h5>
+  <h5 id="job_applicant" data="{{ job_applicant.name }}">Hey {{job_applicant.applicant_name}},
+		You’re applying for the {{job_applicant.one_fm_designation}} position.</h5>
+	<h5>Enter all the details about your career in the below form and tell us how great you`re?</h5>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.html]: This should be as "how great you are!"

File: one_fm/templates/pages/career_history.js [hunk 2]
```diff
@@ -19,15 +19,15 @@ career_history = Class.extend({
       var promotion_select = $(`.promotion_select_${company_no}${promotion_no}`).val()
       var promotion_details_html = "";
       var promotion_details_next_job_title_html = `<div class="my-5 col-lg-6 col-md-6">
-          <label class="form-label">So, Tell me to which position you got promoted first</label>
+          <label class="form-label">So, tell me to which position you got promoted first</label>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "So, tell us what position you got promoted to first."

File: one_fm/templates/pages/career_history.js [hunk 3]
```diff
@@ -19,15 +19,15 @@ career_history = Class.extend({
...
           <input type="text" class="form-control position_${company_no}${promotion_no}" placeholder="Enter Your New Job Title"/>
         </div>`;
       var promotion_details_salary_html = `<div class="my-5 col-lg-6 col-md-6">
-          <label class="form-label">What is Your Increased Salary in KWD?</label>
+          <label class="form-label">What is your increased salary in KWD?</label>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "How much was your incremented salary in KWD?"

File: one_fm/templates/pages/career_history.js [hunk 4]
```diff
@@ -19,15 +19,15 @@ career_history = Class.extend({
...
           <input type="text" class="form-control salary_${company_no}${promotion_no}" placeholder="Enter your increased Salary in KWD"/>
         </div>`;
       var promotion_details_promotion_date_html = `<div class="my-5 col-lg-6 col-md-6">
-          <label class="form-label">When Do you got Promoted?</label>
+          <label class="form-label">When do you got promoted?</label>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "When did you get promoted?"

File: one_fm/templates/pages/career_history.js [hunk 5]
```diff
@@ -65,12 +65,12 @@ career_history = Class.extend({
...
     var next_promotion_details_html = `<div class="col-lg-12 col-md-12 promotion_section_${company_no}${promotion_no}">
-        <label  class="form-label">Did You Got any Promotion or Salary Increase?</label>
+        <label  class="form-label">Did you get any promotion or salary increase?</label>
           <select class="custom-select promotion_select_${company_no}${promotion_no}">
             <option value="0">No, I did not get any promotion or salary increase</option>
-            <option value="1">Yes, I Got a Promotion with a Salary Increase</option>
-            <option value="2">Yes, Only Got a Promotion</option>
-            <option value="3">Yes, Only Got a Salary Increase</option>
+            <option value="1">Yes, I got a promotion with a salary increase</option>
+            <option value="2">Yes, only got a promotion</option>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "Yes, I only got a promotion"

File: one_fm/templates/pages/career_history.js [hunk 6]
```diff
@@ -65,12 +65,12 @@ career_history = Class.extend({
...
+            <option value="3">Yes, only got a salary increase</option>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "Yes, I only got a salary increase"

File: one_fm/templates/pages/career_history.js [hunk 7]
```diff
@@ -137,13 +137,13 @@ career_history = Class.extend({
   },
   create_company_section_html: function(company_no) {
     var company_section_html = `<div class="row col-lg-12 col-md-12 mb-12 company_${company_no}">
-      <h3>So {{job_applicant.applicant_name}}, Tell me about your ${stringifyNumber(company_no)} Company you worked on?</h3>
+      <h3>So {{job_applicant.applicant_name}}, tell me about the ${stringifyNumber(company_no)} company you worked on?</h3>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "So {applicant}, tell us about the {company_no} company you worked for!" (no question mark)

File: one_fm/templates/pages/career_history.js [hunk 8]
```diff
@@ -152,16 +152,16 @@ career_history = Class.extend({
         </select>
       </div>
       <div class="mb-3 col-lg-6 col-md-6">
-        <label class="form-label">When Do you Joined the company</label>
+        <label class="form-label">When do you join the company?</label>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "When did you join the company?"

File: one_fm/templates/pages/career_history.js [hunk 9]
```diff
@@ -152,16 +152,16 @@ career_history = Class.extend({
...
       </div>
       <div class="mb-3 col-lg-6 col-md-6">
-        <label class="form-label">What was your first salary at this company ?</label>
+        <label class="form-label">What was your first salary at this company?</label>
         <input type="text" class="form-control salary_company${company_no}" placeholder="Enter your Salary in KWD"/>
       </div>
 
       <div class="col-lg-12 col-md-12">
-        <label for="Responisbilities" class="form-label">What are your top 3 Responisbilities?</label>
+        <label for="Responisbilities" class="form-label">What are your top 3 responsibilities?</label>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/career_history.js]: "What were your top three responsibilities?"

File: one_fm/templates/pages/job_application.html [hunk 10]
```diff
@@ -9,7 +9,7 @@
 {% endblock %}
 
 {% block header %}
-  <h5 id="job_opening" data="{{ job_opening.name }}">Dear Applicant for the postion of {{ job_opening.designation }}, Enter your Full Name in English.</h5>
+  <h5 id="job_opening" data="{{ job_opening.name }}">Dear applicant for the postion of the {{ job_opening.designation }}, enter your full name in english.</h5>
```
Comment (Ehtesham599) [inline one_fm/templates/pages/job_application.html]: "Dear applicant, for the position {designation}, enter your full name in English."

File: one_fm/templates/pages/job_application.js [hunk 11]
```diff
@@ -63,13 +63,13 @@ job_application = Class.extend({
   show_country: function(applicant_name) {
     if(applicant_name){
       $(".applicant_country").empty();
-      $(".applicant_country").append(`<h5>Hey ${applicant_name} !, Select your Country.</h5>`)
+      $(".applicant_country").append(`<h5>Hey ${applicant_name}, tell me about your nationality.</h5>`)
```
Comment (Ehtesham599) [inline one_fm/templates/pages/job_application.js]: "Hey {applicant}, whats your nationality?"

File: one_fm/templates/pages/job_application.js [hunk 12]
```diff
@@ -63,13 +63,13 @@ job_application = Class.extend({
...
       $(".country_list").removeClass('hide');
     }
   },
   show_applicant_contact_details: function() {
     $(".applicant_contact").empty();
-    $(".applicant_contact").append(`<h5>Enter your mobile number and the email ID to connect you.</h5>`)
+    $(".applicant_contact").append(`<h5>Enter your mobile number and the email id to contact you.</h5>`)
```
Comment (Ehtesham599) [inline one_fm/templates/pages/job_application.js]: "Please provide your email id and mobile number so we could contact you."

File: one_fm/templates/pages/job_application.js [hunk 13]
```diff
@@ -110,7 +110,7 @@ job_application = Class.extend({
   },
   show_cv_section: function(applicant_name) {
     $(".applicant_cv").empty();
-    $(".applicant_cv").append(`<h5>So ${applicant_name}, Attach your CV here to know more about you and then Submit.</h5>`)
+    $(".applicant_cv").append(`<h5>So ${applicant_name}, Attach your CV here to know more about you and then submit it.</h5>`)
```
Comment (Ehtesham599) [inline one_fm/templates/pages/job_application.js]: "Dear {applicant}, please attach your CV here so we can get to know you better."

File: one_fm/templates/pages/job_application.js [hunk 14]
```diff
@@ -138,15 +138,15 @@ job_application = Class.extend({
           btn: this,
           callback: function(r){
             frappe.unfreeze();
-            frappe.msgprint(frappe._("Succesfully Submitted your Job Application and our HR team will be responding to you soon."));
+            frappe.msgprint(frappe._("Successfully Submitted your Job Application and our HR team will be responding to you soon."));
```
Comment (Ehtesham599) [inline one_fm/templates/pages/job_application.js]: "Successfully submitted your application. Our HR team will be responding to you soon."

**Review by Ehtesham599 — COMMENTED** [review]
@pjamsheer Requesting changes.

File: one_fm/templates/pages/job_application.js (hunk 14 above)
Comment (mymi14s) [reply one_fm/templates/pages/job_application.js]: @pjamsheer can you please provide the 
 sentences used as a list, we will collaboratively correct it.

**Review by Ehtesham599 — APPROVED** [review]
LGTM

**Pull request merged**